app/Services/QuizService.php

```diff
@@ -33,4 +33,9 @@ public function getAllQuizzes()
     {
         return Quiz::all();
     }
+
+    public function getQuizByLanguage($languageId)
+    {
+        return Quiz::where('language_id', $languageId)->with('questions.answers')->get();
```

Comment:
Quiz::query()->where('language_id', $languageId)->with('questions.answers')->get();
 
 add the query()